espanol/bailar/abucheo/abucheo.md

```diff
@@ -0,0 +1 @@
+Entonces el Consejero le partio la guitarra en la cabeza a Beatito, para que tenga y guarde, y sea mas respetoso de sus mayores.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.